src/Form/Field/StatsField.php

```diff
@@ -17,27 +17,26 @@
  */
 class StatsField extends FieldClass
 {
-	/**
-	 * Array or query string for reports. Each report needs a `label` and `value` and can have additional `info`, `link`, `icon` and `theme` settings.
-	 */
-	protected array|string $reports;
-
-	/**
-	 * The size of the report cards. Available sizes: `tiny`, `small`, `medium`, `large`
-	 */
-	protected string $size;
-
 	/**
 	 * Cache for the Stats UI component
 	 */
 	protected Stats $stats;
 
-	public function __construct(array $params)
-	{
-		parent::__construct($params);
-
-		$this->reports = $params['reports'] ?? [];
-		$this->size    = $params['size']    ?? 'large';
+	public function __construct(
+		/**
+		 * Array or query string for reports.
+		 * Each report needs a `label` and `value` and can have
+		 * additional `info`, `link`, `icon` and `theme` settings.
+		 */
```

Comment:
Not sure actually if comments like these per argument work and can be parsed by our docs reflections. For reading the code, I think the would be superior to putting them all together in the method docblock.